demos/qml/tutorial_qft_arithmetics.ipynb

```diff
@@ -148,7 +148,7 @@
     "\n",
     "jnp.set_printoptions(precision=3, suppress=True, floatmode='maxprec_equal')\n",
     "\n",
-    "dev = qml.device(\"lightning.qubit\", wires=3, shots=1)\n",
+    "dev = qml.device(\"lightning.qubit\", wires=3)\n",
```

Comment:
@dime10 is there any concern about this change to the demo?

Reply:
I suppose not, indeed using `qml.state` with `shots=1` makes no sense.

I'm assuming verification is now raising errors for these, since `qml.state` will not (or at least shouldn't) appear in the finite-shots section of the toml file?

Reply:
I noticed the last cell in this notebook uses `qml.probs` with `shots=1`, which I think also makes no sense. Could you update that as well?

Reply:
>I'm assuming verification is now raising errors for these

Yes, exactly! For `probs` and `shots=1` it doesn't, since it's a mathematically valid request, just not a very useful one 😅 Thanks for catching that one - I'll update the demo!